app/main.py

```
@@ -1 +1,21 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    source_file_name = parts[1]
+    destination_file_name = parts[2]
+
+    if source_file_name == destination_file_name:
+        return
+
+    try:
+        with (
+            open(source_file_name, "r") as file_in,
+            open(destination_file_name, "w") as file_out,
+        ):
+            for line in file_in:
+                file_out.write(line)
+    except Exception:
```

Catching the base `Exception` (`except Exception:`) is too broad and can hide unexpected errors. Prefer catching specific exceptions such as `FileNotFoundError`, `PermissionError`, or `IsADirectoryError`. This is a recommended improvement so failures are handled explicitly rather than silently swallowed.